lib/oban/queue/producer.ex

```diff
@@ -201,11 +203,18 @@ defmodule Oban.Queue.Producer do
   end
 
   defp schedule_refresh(%State{} = state) do
-    timer = Process.send_after(self(), :refresh, state.refresh_interval)
+    timer = Process.send_after(self(), :refresh, jitter(state.refresh_interval))
 
     %{state | refresh_timer: timer}
   end
 
+  defp jitter(time) do
+    deviation = :rand.uniform() * @max_refresh_deviation * time
```

Comment:
This is the second or third jitter implementation we have sprinkled around. What do you think about adding a helper function for it to Breaker?

Reply:
Great point, I didn't notice. I added a jitter implementation on Broker and changed to use it everywhere. I tested other implementations and this one was slightly faster.